LICENSE.rst

```diff
@@ -0,0 +1,42 @@
+*******
+LICENSE
+*******
+::
+
+    Copyright (c) 2010-2013, University of Wisconsin Computational Nuclear Engineering Research Group
+     All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+
+      - Redistributions of source code must retain the above copyright notice,
+        this list of conditions and the following disclaimer.
+
+      - Redistributions in binary form must reproduce the above copyright
+        notice, this list of conditions and the following disclaimer in the
+        documentation and/or other materials provided with the distribution.
+
+      - Neither the name of the University of Wisconsin Computational
+        Nuclear Engineering Research Group nor the names of its
+        contributors may be used to endorse or promote products derived
+        from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+    AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+    IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+    ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+    LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+    CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+    SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+    INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+    CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+    ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+
+Other Files
+===========
+
+`utils/upload_nuc_data.py` was largely taken from PyNE and maintains its
+ `license <https://github.com/pyne/pyne/blob/develop/license.txt>`_ and
+ copyright: `Copyright 2011-2015, the PyNE Development Team. All rights
```

Comment:
ok, its done

On Mon, May 11, 2015 at 5:42 PM, Anthony Scopatz <notifications@github.com>
wrote:

> In LICENSE.rst
> <https://github.com/cyclus/ciclus/pull/63#discussion_r30089941>:
>
> > +    IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
> > +    ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
> > +    LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
> > +    CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
> > +    SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
> > +    INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
> > +    CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
> > +    ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
> > +    POSSIBILITY OF SUCH DAMAGE.
> > +
> > +Other Files
> > +===========
> > +
> > +`utils/upload_nuc_data.py` was largely taken from PyNE and maintains its
> > + `license <https://github.com/pyne/pyne/blob/develop/license.txt>`_ and
> > + copyright: `Copyright 2011-2015, the PyNE Development Team. All rights
>
> On Mon, May 11, 2015 at 5:40 PM Matthew Gidden <notifications@github.com>
> wrote: > +`utils/upload_nuc_data.py` was largely taken from PyNE and
> maintains its > + `license <
> https://github.com/pyne/pyne/blob/develop/license.txt>`_ and > +
> copyright: `Copyright 2011-2015, the PyNE Development Team. All rights are
> you sure, cyclus/README.rst uses them
> Yes, I am pretty sure because it is super annoying and bothers me
> everytime :).
> http://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#inline-markup-and-special-characters-e-g-bold-italic-verbatim
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/ciclus/pull/63/files#r30089941>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192